locales/en.json

```diff
@@ -1038,6 +1038,9 @@
     "walletAnswer": "Wallets are used to send, receive, store, and display digital assets",
     "walletLink": "Learn about wallets",
     "walletHeading": "Connect Wallet",
+    "walletDetails": "Your Wallet",
+    "walletAccount": "My {0} Account",
+    "total": "Total",
```

Comment:
we already have a "Total" translation

Reply:
Yes, but the `total` is in the group of "spotlight". I think we should not mix them. WDYT
Or I could make a common key "common.total" for both of them for sure.

<img width="366" alt="image" src="https://user-images.githubusercontent.com/31397967/215532893-765cd932-9440-4524-aa5b-aa8a705d375c.png">

Reply:
I already had this case before and I preferred to use already declared key even if it is not in the same group since some of them have already been translated.
but it's sure that in the end we would need a better naming convention

Reply:
It makes sense. I have updated.